secant/UI Components/Chips/EnumeratedChip.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  EnumeratedChip.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/19/21.
+//
+
+import SwiftUI
+
+struct EnumeratedChip: View {
+    @Clamping(1...24)
+    var index: UInt = 1
+
+    var text: String
+
+    var body: some View {
+        NumberedText(
+            number: index,
+            text: text
+        )
+        .frame(
+            minWidth: 0,
+            maxWidth: .infinity,
+            minHeight: 30,
+            maxHeight: .infinity,
+            alignment: .leading
+        )
+        .padding(.horizontal, 16)
+        .padding(.vertical, 8)
+
+        .background(
+            Asset.Colors.Buttons.primaryButtonPressed.color
+        )
+        .cornerRadius(6)
+    }
+}
+
+struct NumberedText: View {
+    @Clamping(1...24) var number: UInt = 1
+
+    var text: String
+
+    @ViewBuilder var numberedText: some View {
+        Text(number.superscriptRepresentation)
+            .foregroundColor(
+                Asset.Colors.Text.highlightedSuperscriptText.color
+            )
+            .font(
```

Comment:
Is there anything we can do with font sizing to make it so that the words don't wrap when used on a smaller phone?